### PR TITLE
Doc fix

### PR DESCRIPTION
**Pull request by jpic**

### CODE REVIEW

**Comment by max-arnold** [comment]
I cleaned it up a bit to reflect changes in #111. Below is the build log (note the WARNING file:///home/jpic/env/src/autocomplete-light/docs/build/html/objects.inv):

```
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v1.3.6
loading pickled environment... done
loading intersphinx inventory from file:///home/jpic/env/src/autocomplete-light/docs/build/html/objects.inv...
WARNING: intersphinx inventory 'file:///home/jpic/env/src/autocomplete-light/docs/build/html/objects.inv' not fetchable due to <class 'urllib2.URLError'>: <urlopen error [Errno 2] No such file or directory: '/home/jpic/env/src/autocomplete-light/docs/build/html/objects.inv'>
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 0 source files that are out of date
updating environment: 0 added, 1 changed, 0 removed
reading sources... [100%] index
../README.rst:None: WARNING: nonlocal image URI found: https://secure.travis-ci.org/yourlabs/django-cities-light.png?branch=master
../README.rst:None: WARNING: nonlocal image URI found: https://pypip.in/d/django-cities-light/badge.png
../README.rst:None: WARNING: nonlocal image URI found: https://pypip.in/v/django-cities-light/badge.png
../README.rst:None: WARNING: nonlocal image URI found: https://codecov.io/github/yourlabs/django-cities-light/coverage.svg?branch=stable/3.x.x
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index
generating indices... genindex py-modindex
highlighting module code... [100%] cities_light.admin
writing additional pages... search
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 5 warnings.

Build finished. The HTML pages are in build/html.
```

NOTE: you need to refetch this branch since I modified its history.

File: README.rst [hunk 1]
```diff
@@ -74,6 +74,17 @@ This command is well documented, consult the help with::
 
     ./manage.py help cities_light
 
+Development
+-----------
+
+To build the docs use the following steps:
+
+1. mkvirtualenv dcl-doc
```
Comment (jpic) [inline README.rst]: That's a command that wraps around both virtualenv and source ../activate ?

**Comment by jpic** [comment]
LGTM !

Do you want to merge it @max-arnold ?

File: README.rst (hunk 1 above)
Comment (max-arnold) [inline README.rst]: Yep! Can't recommend virtualenvwrapper highly enough!

Comment (jpic) [inline README.rst]: Automatic virtualenv activation is pretty cool too ;)

http://blog.yourlabs.org/post/21015702927/automatic-virtualenv-activation

**Comment by max-arnold** [comment]
I'm happy to have this privilege! :)

**Pull request merged**